src/webui/www/private/scripts/client.js

```diff
@@ -455,7 +455,10 @@ window.addEvent('load', function() {
         const categoryList = $('categoryFilterList');
         if (!categoryList)
             return;
-        categoryList.empty();
+
+        categoryList.getChildren().each(function(c) {
```

Comment:
may i suggest we use `replaceChildren()`.
https://stackoverflow.com/a/65413839

Reply:
It is old enough for us: https://github.com/qbittorrent/qBittorrent/tree/master/src/webui/www#browser-compatibility